.github/workflows/pull-request.yml

```diff
@@ -41,6 +41,13 @@ jobs:
           TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
           NEXT_TELEMETRY_DISABLED: 1
 
+      - name: Run prettier
```

Comment:
Why would we want to run prettier on CI? 🤔

Reply:
To check if there is any code formatting issue.
`npx turbo prettier` only checks that files are already formatted, it doesn't overwrite them.

Reply:
Then the name of the job should not be changed, as it is not formatting the files and still checking the formatting.

Reply:
Can we revert the name of the job, then? prettier is part of the linting checks